doc/rados/operations/data-placement.rst

```diff
@@ -2,40 +2,45 @@
  Data Placement Overview
 =========================
 
-Ceph stores, replicates and rebalances data objects across a RADOS cluster
-dynamically.  With many different users storing objects in different pools for
-different purposes on countless OSDs, Ceph operations require some data
-placement planning.  The main data placement planning concepts in Ceph include:
+Ceph stores, replicates, and rebalances data objects across a RADOS cluster
+dynamically. Because different users store objects in different pools for
+different purposes on many OSDs, Ceph operations require a certain amount of
+data- placement planning. The main data-placement planning concepts in Ceph
+include:
 
-- **Pools:** Ceph stores data within pools, which are logical groups for storing
-  objects. Pools manage the number of placement groups, the number of replicas,
-  and the CRUSH rule for the pool. To store data in a pool, you must have
-  an authenticated user with permissions for the pool. Ceph can snapshot pools.
-  See `Pools`_ for additional details.
+- **Pools:** Ceph stores data within pools, which are logical groups used for
+  storing objects. Pools manage the number of placement groups, the number of
+  replicas, and the CRUSH rule for the pool. To store data in a pool, it is
+  necessary to be an authenticated user with permissions for the pool. Ceph is
+  able to make snapshots of pools. For additional details, see `Pools`_.
 
-- **Placement Groups:** Ceph maps objects to placement groups (PGs).
-  Placement groups (PGs) are shards or fragments of a logical object pool
-  that place objects as a group into OSDs. Placement groups reduce the amount
-  of per-object metadata when Ceph stores the data in OSDs. A larger number of
-  placement groups (e.g., 100 per OSD) leads to better balancing. See
-  :ref:`placement groups` for additional details.
+- **Placement Groups:** Ceph maps objects to placement groups. Placement
+  groups (PGs) are shards or fragments of a logical object pool that place
+  objects as a group into OSDs. Placement groups reduce the amount of
+  per-object metadata that is necessary for Ceph to store the data in OSDs. A
+  greater number of placement groups (for example, 100 PGs per OSD as compared
+  with 50 PGs per OSD) leads to better balancing. For additional details, see
+  :ref:`placement groups`.
 
-- **CRUSH Maps:**  CRUSH is a big part of what allows Ceph to scale without
-  performance bottlenecks, without limitations to scalability, and without a
-  single point of failure. CRUSH maps provide the physical topology of the
-  cluster to the CRUSH algorithm to determine where the data for an object
-  and its replicas should be stored, and how to do so across failure domains
-  for added data safety among other things. See `CRUSH Maps`_ for additional
-  details.
+- **CRUSH Maps:**  CRUSH plays a major role in allowing Ceph to scale while
+  avoiding certain pitfalls, such as performance bottlenecks, limitations to
```

Comment:
s/such as/including/

I hate "such as" so hard.  Kinda like "things like X and Y" which connotes things that are LIKE X and Y but not X and Y themselves.

Yeah I'm like that.

Reply:
@anthonyeleven - In this case, the reason I went with "such as" rather than "including" is to avoid even the momentary suggestion that _including performance bottlenecks_ is one of the pitfalls. Here's an alternative you might prefer to the current version: _avoiding certain pitfalls (for example, performance bottlenecks, limitations to scalability, and single points of failure)_.